src/test/common/fdc3.user-channels.ts

```diff
@@ -176,11 +178,9 @@ export function createUserChannelTests(cc: ChannelControl<any, any>, documentati
 
         await cc.joinChannel(channels[0]);
         await cc.joinChannel(channels[1]);
-        await cc.openChannelApp(scTestId8, channels[0].id, JOIN_AND_BROADCAST);
-        await wait();
+        await cc.openChannelApp(scTestId8, channels[0].id, JOIN_AND_BROADCAST, undefined, true, contextId);
+        await wait(3000);
```

Comment:
fyi this isn't necessary as the parameter passed to `wait()` defaults to the value of `constants.WaitTime` if left empty